utils/mock-data-loader/gef/eligibilityCriteria.js

```diff
@@ -8,7 +8,7 @@ const ELIGIBILITY_CRITERIA = [
       {
         id: 12,
         name: 'coverStart',
-        text: 'The period between the Cover Start Date and the Cover End Date does not exceed the Facility Maximum Cover Period.',
+        text: 'The period between the Cover Start Date and the Cover End Date does not exceed: the Facility Maximum Cover Period.',
```

Comment:
@abhi-markan I think we've accidentally changed the GEF ECs as well (here and in some of the test deals/ECs)